src/Apps/Order/redirects.tsx

```diff
@@ -101,6 +102,19 @@ const goToReviewIfOrderIsPending: RedirectPredicate = ({ order }) => {
   }
 }
 
+const goToRespondIfMyLastOfferIsNotMostRecentOffer: RedirectPredicate = ({
+  order,
+}) => {
+  if (
+    order.myLastOffer &&
+    order.lastOffer &&
+    moment(order.myLastOffer.createdAt).isAfter(order.lastOffer.createdAt)
```

Comment:
Instead of comparing timestamps, should we just verify that the `id`s on these offers are the same?

Reply:
🤔 I think they shouldn't be the same. If the user makes a counteroffer they will be different. Also they will be different before the buyer makes a counteroffer because the buyer made an initial offer at some point. So I think we need to check that the current `myLastOffer` came after the current `lastOffer`

Reply:
Ohhh duh you're totally right. Don't know where I was this morning 😅